embedly/embedly/extract.py

```diff
@@ -23,7 +27,12 @@ def _get_cached_url(self, url):
         cached_data = self.redis_client.get(cache_key)
 
         if cached_data is not None:
-            return json.loads(cached_data)
+            try:
+                return json.loads(cached_data)
+            except ValueError:
+                raise URLExtractorException(
```

Comment:
If we're unable to read the data for a URL from the cache, we can theoretically continue handling this request by ignoring the cache failure and continuing to query embedly for the URL.